Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -1343,23 +1343,8 @@ void MainWindow::showSceneContextMenu(const QPoint& p) {
         QMenu menu;
         Q_FOREACH(QString name, menu_actions.keys())
         {
-          menu.addAction(name,
-                         [this, name]()
-          {
-            Q_FOREACH(Scene::Item_id id, scene->selectionIndices())
-            {
-              Scene_item* item = scene->item(id);
-              Q_FOREACH(QAction* action, item->contextMenu()->actions())
-              {
-                if(action->text() == name)
-                {
-                  action->trigger();
-                  break;
-                }
-              }
-            }
-          }
-          );
```

Comment:
Indeed, the overload of `QMenu::addAction()` overload taking a functor was added in Qt 5.6. But there is no reason not to use a lambda expression for the connect:
```C++
QAction* action = menu.addAction(name);
connect(action, &QAction::triggered, this,
        [this, name]()
        {
          Q_FOREACH(Scene::Item_id id, scene->selectionIndices())
          {
            Scene_item* item = scene->item(id);
            Q_FOREACH(QAction* action, item->contextMenu()->actions())
            {
              if(action->text() == name)
              {
                action->trigger();
                break;
              }
            }
          }
        }
        );
```